packages/node/src/services.ts

```diff
@@ -46,6 +46,10 @@ export class FirebaseServiceFactory {
   createStoreService(storeServiceKey: string): IStoreService {
     return new FirebaseStoreService(this.app.database(), storeServiceKey);
   }
+
+  async close() {
```

Comment:
Maybe `closeConnection` or `disconnect` or `disconnectAndDeleteHistory` (does it delete history?)